content/blog/2022/11/2022-11-14-hacktoberfest-recap.adoc

```diff
@@ -0,0 +1,159 @@
+These logos help connect Jenkins to users globally and extend the community even further.
+Thea is a freelance developer, and this is their first time contributing to Jenkins in Hacktoberfest.
+Their motivation when starting was curiosity and learning more about open source.
+Through their contributions, they have learned how to work with someone else's code.
+There have been challenges when new tools are installed, but don't work as expected.
+However, these are learning experiences, as each one is an opportunity to understand what is causing the behavior.
+
+image:/images/post-images/2022-11-14-hacktoberfest-recap/thea.png[Thea Mushambadze, width=20%]
+
+Next, we want to introduce link:https://github.com/OmkarBorhade98[Omkar Borhade] from Pune, India.
+Omkar just started as a software engineer, after graduating as a Mechanical Engineer.
+They are new to open source, and this was their first time participating in Hacktoberfest.
+Their goal was to become more familiar the Jenkins project.
+This was acheived by a lot of reading, approaching issues with an open mind, and providing documentation.
+As a new contributor, they shared it can be difficult to understand the issue and what is needed for resolution.
+This knowledge comes with experience, so have some patience when getting started.
+
+image:/images/post-images/2022-11-14-hacktoberfest-recap/omkar.png[Omkar Borhade, width=20%]
+
+link:https://github.com/krisstern[Kris Stern] is a full-stack software engineer based out of Hong Kong SAR, China.
+They have previously trained as an observational astrophysicist with a PhD in physics, and is currently studying part-time for a computer science master's degree. 
+This is their third Hacktoberfest, and first Hacktoberfest contributing to Jenkins.
+They have contributed to open source previously for astronomy, but shifted focus since graduating.
+Their goals during Hacktoberfest were giving back to the Jenkins community and acquiring more software development skills along the way.
+
+
+
+== Returning Jenkins contributors
+
+First, we want to welcome back link:https://github.com/jimklimov[Jim Klimov].
+Jim is based in the Czech Republic, and has been working in open source for quite some time.
+They have worked on various enterprise document workflow and messaging systems, and is currently a DevOps architect.
+They are the creator and maintainer of several Jenkins plugins.
+Along with their own experience, they learned a lot from the people that came before.
+Having this knowledge feels like standing on the shoulders of giants.
+This is their second Hacktoberfest in Jenkins, but contributes all year round.
+Challenges that have come up for Jim include maintainability expectations to get pull requests merged, and architecture becoming more costly as time goes on.
```

Comment:
:) That's a very nice way of saying that Mark Waite is very slow to review his major pull request to the git plugins.  Jim is very kind!